parts/k8s/cloud-init/masternodecustomdata.yml

```diff
@@ -461,6 +461,9 @@ MASTER_CONTAINER_ADDONS_PLACEHOLDER
 {{- if EnableDataEncryptionAtRest }}
     sed -i "s|<etcdEncryptionSecret>|\"{{WrapAsParameter "etcdEncryptionKey"}}\"|g" /etc/kubernetes/encryption-config.yaml
 {{end}}
+    {{- /* Ensure that container traffic can't connect to internal Azure IP endpoint */}}
+    iptables -I OUTPUT -s {{GetClusterSubnet}} -d 168.63.129.16 -p tcp --dport 80 -j DROP
```

Comment:
Is the intent here to drop all traffic to wireserver from the host? Pods running with host network have the node IP as pod IP and this rule would just block all traffic.

Reply:
Also, this should be same for v4/v6 or dual-stack clusters because we use the host v4 networking for dns, etc..

Reply:
nope, we should not block processes like guest agent running in the vm. this is the part i was struggling to differentiate: how to block only container traffic whose source IP is node IP?